library/pom.xml

```diff
@@ -372,9 +377,39 @@
 							</execution>
 						</executions>
 					</plugin>
+
 				</plugins>
 			</build>
 		</profile>
+		<profile>
```

Comment:
This seems kind of a cool idea.

There's a couple things it's fine to just have in the ckeditor.launch.js (won't cause an error if they're not present) for sure the value in the toolbar
['A11ychecker']

Just put that in there (for ATD it already is in there). I think the propreties should be more specific so in-case we add other dynamic plugins in the future (there are a few other GPL ones) we can have them add themselves similarly to this one. 

Maybe the activation should just be via property like

```
    <activation>
      <property>
        <name>ckeditor-extras</name>
      </property>
    </activation>
```

Then we can activate other ones too?